mlir/lib/Dialect/Affine/Transforms/NormalizeMemRefs.cpp

```diff
@@ -0,0 +1,220 @@
+//===- NormalizeMemRefs.cpp ---------------------------------------===//
```

Comment:
This pass can actually live in `lib/Transforms/` instead of in `lib/Dialects/Affine/Transforms` because it isn't specific to the affine dialect.